utils/node_local_test.py

```diff
@@ -114,6 +118,20 @@ def set_args(self, args):
     def __getitem__(self, key):
         return self.bc[key]
 
+    def flush_bz2(self):
+        if not self._compress_proc:
+            return
+        self.lt_compress.start()
+        self._compress_proc.wait()
+        self.lt_compress.stop()
+        self._compress_proc = None
+
+    def compress_file(self, filename):
+        """Compress a file using bz2 for space reasons"""
+
+        self.flush_bz2()
+        self._compress_proc = subprocess.Popen(['bzip2', '--best', filename])
```

Comment:
(pylint-consider-using-with) Consider using 'with' for resource-allocating operations